internal/stack/certs.go

```diff
@@ -140,9 +145,18 @@ func initServiceTLSCertificates(ca *certs.Issuer, caCertFile string, certFile, k
 		return certs.LoadCertificate(certFile, keyFile)
 	}
 
-	cert, err := ca.Issue(certs.WithName(service))
-	if err != nil {
-		return nil, fmt.Errorf("error initializing certificate for %q", service)
+	var cert *certs.Certificate
+	var err error
+	if service == "elastic-agent" {
+		cert, err = ca.IssueClient(certs.WithName(service))
+		if err != nil {
+			return nil, fmt.Errorf("error initializing certificate for %q", service)
+		}
```

Comment:
Why `elastic-agent` service requires this change ?

Reply:
Looking into those options, it is needed to ensure that it is not hit the same error as the one mentioned here: 

https://github.com/elastic/elastic-package/pull/1646/files#diff-9b02d91c4a392a35e7a9ff3e09d1f61d6c7a9a2b42f7034a30b5feaf265ff15cR191
```
			// Required for Chrome in OSX to show the "Proceed anyway" link.
			// https://stackoverflow.com/a/64309893/28855
```

Probably, is this just for the Kibana cert ? @jsoriano

Reply:
This is a `client certificate` and this is because logstash expects the input plugin to be a client and hold a client cert and the validation fails with the existing one as they are server certificates.

Reply:
Logstash is expecting the client certificates even after removing this config.